pkg/aws/ec2/ec2.go

```diff
@@ -268,6 +269,12 @@ func (c *Client) GetVpcs(ctx context.Context) (ipamTypes.VirtualNetworkMap, erro
 			vpc.PrimaryCIDR = *v.CidrBlock
 		}
 
+		for _, c := range v.CidrBlockAssociationSet {
+			if cidr := aws.ToString(c.CidrBlock); cidr != vpc.PrimaryCIDR {
```

Comment:
This I believe won't compile on 1.8, it needs to be
```suggestion
			if cidr := *c.CidrBlock; cidr != vpc.PrimaryCIDR {
```

Reply:
I used `aws.StringValue` just the same as for [the v1.9 backport](https://github.com/cilium/cilium/pull/15331#issuecomment-799249798), should be ok now.